src/cgame/cg_buildable.cpp

```diff
@@ -1129,7 +1129,8 @@ void CG_GhostBuildable( int buildableInfo )
 	vec3_t        mins, maxs;
 	trace_t       tr;
 	float         scale;
-	buildable_t   buildable = (buildable_t)( buildableInfo & SB_BUILDABLE_MASK ); // assumed not BA_NONE
```

Comment:
The assertion should be on `buildable` not `buildableInfo`, right? Also `ASSERT_NQ`

Reply:
Now that I look anew at it, yes, this would make more sense. Looks like buildableInfo is really a "struct" with both the buildable_t and some flags in it...

Reply:
assert not equal

Reply:
Ok. Well, I replaced it with a range anyway, which makes more sense (as long as NUM_BUILDABLE is really the last one). Or there's also a specific assert for that? (that seems a bit overkill to have specific asserts for those kind of stuff though)